src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
+    axios.get('http://localhost:5000/events').then((response) => {
+      for (var key in response.data){
+        const e = response.data[key]
+        const eventTitle = e.event_title;
+        const id = e.id;
+        const numJudges = e.num_judges;
+        const eventDate = e.event_date;
+
         const date = isObject(eventDate)
-          ? new Date(eventDate.seconds * 1000).toLocaleDateString('en-GB')
-          : eventDate;
+        ? new Date(eventDate.seconds * 1000).toLocaleDateString('en-GB')
```

Comment:
can you add a `TODO` here to remove the `isObject` check

Reply:
Fixed in https://github.com/uwblueprint/dancefest-web/pull/94/commits/d85709a13e9329b593ed129062f9ec252ba6fa31, thanks!

Reply:
No longer needed.
IsObject check is removed in https://github.com/uwblueprint/dancefest-web/pull/94/commits/14df67de75375d7d3c265cac721d763d08a9eb69
Date formatting has also been fixed in https://github.com/uwblueprint/dancefest-web/pull/94/commits/6627d1f55e4228c4e1419ce1aa2dfbbd222ecbef